src/app/docs/button/button-docs.component.html

```diff
@@ -0,0 +1,196 @@
+            </p>
+          </td>
+        </tr>
+        <tr>
+          <td class="sky-docs-table-cell">
+            <button type="button" class="sky-btn sky-btn-link">Link</button>
+          </td>
+          <td class="sky-docs-table-cell">
+            <p>
+              Usage
+            </p>
+            <p>
+              Link buttons trigger actions that take users to other pages or actions that end tasks or exit modals (such as <strong>Cancel</strong>).
+            </p>
+          </td>
+          <td class="sky-docs-table-cell">
+            <p>
+              Code
+            </p>
+            <p>
+              <sky-code>&lt;button type=&quot;button&quot; class=&quot;sky-btn sky-btn-link&quot;&gt;Link button&lt;/button&gt;</sky-code>
+            </p>
+          </td>
+        </tr>
+        <tr>
+          <td class="sky-docs-table-cell">
+            <button type="button" class="sky-btn sky-btn-link-inline">Inline link</button>
+          </td>
+          <td class="sky-docs-table-cell">
+            <p>
+              Usage
+            </p>
+            <p>
+              Inline link buttons trigger the same type of actions as link buttons, but the styling is altered to fit inline.
+            </p>
+          </td>
+          <td class="sky-docs-table-cell">
+            <p>
+              Code
+            </p>
+            <p>
+              <sky-code>&lt;button type=&quot;button&quot; class=&quot;sky-btn sky-btn-link-inline&quot;&gt;Inline link&lt;/button&gt;</sky-code>
+            </p>
+          </td>
+        </tr>
+        <tr>
+          <td class="sky-docs-table-cell">
+            <div style="margin-bottom: 10px;">
+              <button type="button" class="sky-btn sky-btn-primary sky-btn-disabled">Primary</button>
+            </div>
+            <div style="margin-bottom: 10px;">
+              <button type="button" class="sky-btn sky-btn-default sky-btn-disabled">Secondary</button>
+            </div>
+            <div style="margin-bottom: 10px;">
+              <button type="button" class="sky-btn sky-btn-link sky-btn-disabled">Link</button>
+            </div>
+          </td>
+          <td class="sky-docs-table-cell">
+            <p>
+              Usage
+            </p>
+            <p>
+              The disable class disables all button types. The most common use case is to disable actions until users complete prerequisite tasks.
+            </p>
+          </td>
+          <td class="sky-docs-table-cell">
+            <p>
+              Code
+            </p>
+            <p>
+              <sky-code>class="sky-btn-disabled"</sky-code>
+            </p>
+          </td>
+        </tr>
+      </table>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="UX guidelines"
+    >
+      <sky-docs-design-guideline
+        heading="Button text"
```

Comment:
I don't think we need this heading if we change the parent header to `Content`. We needed `Button text` in the current docs because `UX guidelines` is such a catch-all label, so we needed a secondary header to indicate what the content is going to be. But with the `Content` header, we are following our new pattern for section headers, so I only think we need subheaders if we have multiple subsections, which we don't here.